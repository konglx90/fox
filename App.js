import App from './src/app';

import { Sentry } from 'react-native-sentry';

const sentryDsn = Platform.select({"ios":"https://44587f361c654dac89a556228bd0e0dd:9b16e4fe3c224d18b1ae40a4e1a69f9c@sentry.io/273686","android":"https://44587f361c654dac89a556228bd0e0dd:9b16e4fe3c224d18b1ae40a4e1a69f9c@sentry.io/273686"});
Sentry.config(sentryDsn).install();


export default App;
