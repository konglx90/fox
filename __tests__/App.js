import 'react-native';
import React from 'react';
import App from '../App';
import KButton from '../src/components/common/button';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <App />
  );
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer
    .create(<KButton />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
