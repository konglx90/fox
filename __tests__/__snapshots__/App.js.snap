// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  loading...
</Text>
`;

exports[`renders correctly 2`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={undefined}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Go to news list
      </Text>
    </View>
  </View>
</View>
`;
